ports/esp32/partitions-app3M_fat9M_fact512k_16MiB.csv

```diff
@@ -0,0 +1,10 @@
+# Notes: the offset of the partition table itself is set in
```

Comment:
This partition layout is very specific to the board, so I think better suited to live in the board directory.